packages/mdc-shape/_mixins.scss

```diff
@@ -24,11 +24,19 @@
 @import "./functions";
 
 @mixin mdc-shape-radius($radius, $rtl-reflexive: false) {
+  // Even if $rtl-reflexive is true, only emit RTL styles if we can't easily tell that the given radius is symmetrical
+  $needs-flip: $rtl-reflexive and length($radius) > 1;
+
+  @if ($needs-flip) {
+    /* @noflip */
+  }
+
```

Comment:
is annotation with new line acceptable?

this generates

```
/* @noflip */

border-radius: mdc-shape-prop-value($radius);
```

instead of

```
/* @noflip */
border-radius: mdc-shape-prop-value($radius);
```

Reply:
I was wary of that too, but when I looked at the diff between the all-in-one CSS before and after, I didn't see any empty newlines generated by these changes.